packages/@aws-cdk/aws-lambda/README.md

```diff
@@ -339,6 +339,83 @@ const fn = new lambda.Function(this, 'MyFunction', {
 fn.currentVersion.addAlias('live');
 ```
 
+## Function URL
+
+A function URL is a dedicated HTTP(S) endpoint for your Lambda function. When you create a function URL, Lambda automatically generates a unique URL endpoint for you.
+
+Function URLs are dual stack-enabled, supporting IPv4 and IPv6. After you configure a function URL for your function, you can invoke your function through its HTTP(S) endpoint via a web browser, curl, Postman, or any HTTP client. Lambda function URLs use [resource-based policies](https://docs.aws.amazon.com/lambda/latest/dg/access-control-resource-based.html) for security and access control. Function URLs also support cross-origin resource sharing (CORS) configuration options.
+
+You can apply function URLs to any function alias, or to the $LATEST unpublished function version. You can't add a function URL to any other function version.
+
+To create a function URL, use the `lambda.FunctionUrl` construct:
+
+```ts
+const fn = new lambda.Function(this, 'MyLambda', {
+  code: new lambda.InlineCode('hello()'),
```

Comment:
That's not how we do InlineCode (we do `lambda.Code.inline()`), and the example code doesn't work. Just leave the initialization of a lambda function implicit.